mtt-management/src/mtt_types.rs

```diff
@@ -314,8 +314,133 @@ pub fn set_4k_page_type(
 }
 
 // Gets the base address of the MTT established by the trusted platform-FW.
-// TODO: Implement platform specific functionality
+// TODO: Implement platform specific functionality.
 #[cfg(all(target_arch = "riscv64", target_os = "none"))]
 fn get_mtt_base() -> u64 {
     todo!("Implement code to get MTT base address");
 }
+
+// Testing only function intended to emulate functionality
+// to get the MTT base.
+#[cfg(not(all(target_arch = "riscv64", target_os = "none")))]
```

Comment:
Why not make this cfg(test) ?

Reply:
It resulted in a duplicate function definition, and other places in the code follow the same pattern for similar reasons:

```

#[cfg(not(all(target_arch = "riscv64", target_os = "none")))]
unsafe fn ecall_send(_msg: &SbiMessage) -> Result<u64> {
    panic!("ecall_send called");
}

```

Reply:
Ah, fair enough.